pkg/logging/fluentd_logger.go

```diff
@@ -110,21 +117,52 @@ func (f *FluentdLogger) Process(_ string, config *logging.Config) error {
 	if err != nil {
 		return fmt.Errorf("failed to create fluent client: %w", err)
 	}
+	var jsonLogger *logrotate.Logger
```

Comment:
Out of curiosity: Is it not possible to exec the nerdctl log driver or call the go routine for json driver run directl.